alma/controllers/front/capabilities.php

```diff
@@ -49,10 +49,13 @@ public function postProcess()
             $endpoint = Tools::getHttpHost(true)."/module/alma/report";
         }
         $data = array(
-            'webhook' => "share_of_chekout", 
             'platform' => "Prestashop",
-            'version' => _PS_VERSION_,
-            'endpoint' => $endpoint);
+            'version' => _PS_VERSION_,            
+            'webhooks' => array(
+                'webhook' => 'share_of_chekout', 
+                'endpoint' => $endpoint
+                )
+            );
```

Comment:
Change the payload shape to:

```php
            'platform' => array( 'name' => 'Prestashop', 'version' => _PS_VERSION_, 'module_version' => Alma::VERSION),
            'webhooks' => array(
                array(
                    'webhook' => 'share_of_checkout', 
                    'endpoint' => $endpoint
                )
            )
```

Reply:
Can we surface the configured timeout from PHP config as well?
If yes, let's add it to the `'platform'` dict